src/main/java/org/folio/circulation/domain/policy/FixedDueDateSchedules.java

```diff
@@ -46,15 +58,15 @@ static FixedDueDateSchedules from(JsonObject representation) {
 
   private Predicate<? super JsonObject> isWithin(DateTime date) {
     return schedule -> {
-      DateTime from = DateTime.parse(schedule.getString("from"));
-      DateTime to = DateTime.parse(schedule.getString("to"));
+      DateTime from = DateTime.parse(schedule.getString("from")).withZoneRetainFields(timeZone);
```

Comment:
This appears to convert the timezone, without changing the field values, is that correct?

Can you help me understand why this is needed? 

Does this not make these values potentially incorrect? If the schedule has a `from` of 05:00 because it is meant to be midnight in a non-UTC timezone, won't this make this 05:00 in the new timezone?

Reply:
Comparing the due date in the interval of a fixed schedule should take into account the selected time zone since a time zone has already been added to the due date.

Reply:
Hmm, the due dates for fixed due date schedules are specified as date and time already, and therefore should already have timezone information.

If so, IMO it is the responsibility of the client specifying the fixed due date schedules to ensure they are correctly respecting the time zone.

I don't think we should be overriding that decision here. Have I missed something?

@skoczko @kostyantyn-kh What do you think?

Reply:
@marc-johnson It appears that fixed due date schedule effectively stores date-only values. Due to the fact that RMB only has a date-time format, it adds zeroes for time and time zone offset.
My vision is that fixed due date schedules can work as calender do: store date-only values, that are converted to date-time by a client (that is aware of time zone).
Do you have concerns about this approach?

Reply:
According to the [schema)(
https://github.com/folio-org/mod-circulation-storage/blob/5d3caf5a7bb69afec62aedfac2c61733f0d6cbd2/ramls/schedule.json#L6) the schedules define dates and times.

In my opinion, when dates and times are specified with a timezone, they should be respected.

If what the UI / sample records defines for fixed due date schedules is incorrect, now that we have started working with time zones, I think that should be fixed, and not compensated for in the logic in circulation.

If, alternatively, we want to convert the schedules to be dates only, like calendar, then we should plan that change in and explicitly model it that way.

Maybe we need to move this conversation to JIRA so we can include the product owners.

Reply:
@dmytrotur @OleksiiMaksymov Please can you write this up on the JIRA issue for CIRC-238, I think we need to discuss this with a broader audience

Reply:
@marcjohnson-kint Ok, I'll do.

Reply:
@marc-johnson Not raised that question yet. It seems like it can take some time to discuss this. Can we revert this change and merge the pull request for now, and discuss it later due to the Q1?

Reply:
@dmytrotur Sure, I'm happy if this is removed whilst we discuss it further.

And if this surfaces during acceptance testing we can discuss the best resolution on a case by case basis.

Reply:
Revert changes